modules/compute/container-registry/README.md

```diff
@@ -0,0 +1,118 @@
+| `diagnosticEventHubAuthorizationRuleId` | `string` | No       | Resource ID of the diagnostic event hub authorization rule for the Event Hubs namespace in which the event hub should be created or streamed to.                                                                                                                                                                                                                                                     |
+| `diagnosticEventHubName`                | `string` | No       | Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.                                                                                                                                                                                                                                                       |
+| `logsToEnable`                          | `array`  | No       | The name of logs that will be streamed.                                                                                                                                                                                                                                                                                                                                                              |
+| `metricsToEnable`                       | `array`  | No       | The name of metrics that will be streamed.                                                                                                                                                                                                                                                                                                                                                           |
+
+## Outputs
+
+| Name              | Type   | Description                                                        |
+| :---------------- | :----: | :----------------------------------------------------------------- |
+| resourceGroupName | string | The resource group the Azure Container Registry was deployed into. |
+| resourceId        | string | The resource ID of the Azure Container Registry.                   |
+| name              | string | The name of the Azure Container Registry.                          |
+| loginServer       | string | The login server URL of the Azure Container Registry.              |
+
+## Examples
+
+### Example 1
+
+An example of how to deploy Azure Container Registry using the minimum required parameters.
+
+```bicep
+module containerRegistry 'br/public:compute/container-registry:1.0' = {
```

Comment:
I think you're going to need a 3 part version number in the readme.
Deploying a module from the registry with a 2 part version will result in

> Unable to restore the module with reference "MODULEPATH": The module does not exist in the registry.bicep(BCP192) 

`br/public:compute/container-registry:1.0.1`

Reply:
Thanks 👍